llama.cpp

```diff
@@ -2736,6 +2756,8 @@ static struct ggml_cgraph * llm_build_falcon(
         cur = ggml_add(ctx0,
                 ggml_mul(ctx0, cur, model.output_norm),
                 model.output_norm_b);
+        offload_func_nr(cur->src[0]);
+        offload_func_nr(cur);
```

Comment:
Was this intentionally not offloaded, or simply missed? @slaren

Reply:
It is intentional, since the tensor must be accessible for the CPU (this is the embeddings). It's the same in llama graph.